app/views/search/_spatial_manual_entry.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="manual-coord-entry-container" class="manual-coord-entry">
+  <p data-bind="visible: !ui.spatialType.manualEntryVisible() && !ui.spatialType.querySpatial()"><i class="fa fa-fw fa-crop"></i></p>
+  <p data-bind="visible: ui.spatialType.manualEntryVisible() || ui.spatialType.querySpatial()">
+    <label for="manual-coord-entry-point"><!-- ko text: ui.spatialType.displaySpatial() + ':' --><!-- /ko --></label>
+    <!-- ko if: ui.spatialType.displaySpatial() == 'Point' -->
+    <input type="text" id="manual-coord-entry-point" name="manual-coord-entry-point" data-bind="value: spatialEntry.point, valueUpdate: 'blur'" placeholder="lat, lon (e.g. 44.2, 130)">
+    <!--/ko-->
+    <p>
+    <!-- ko if: ui.spatialType.displaySpatial() == 'Rectangle' && ui.spatialType.manualEntryVisible() -->
+    <label for="manual-coord-entry-swpoint">Southwest Point</label>
```

Comment:
Just a quick trick to avoid having to set `id` and `for` attributes on labels and inputs, this is valid HTML that does the same thing:

    <label>Southwest Point <input type="text" data-bind="value: spatialEntry.swPoint...."></label>